dataland-e2etests/src/test/kotlin/org/dataland/e2etests/CompanyDataControllerTest.kt

```diff
@@ -4,48 +4,88 @@ import org.dataland.datalandbackend.openApiClient.api.CompanyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.api.EuTaxonomyDataControllerApi
 import org.dataland.datalandbackend.openApiClient.api.MetaDataControllerApi
 import org.dataland.datalandbackend.openApiClient.model.CompanyAssociatedDataEuTaxonomyData
-import org.dataland.datalandbackend.openApiClient.model.CompanyMetaInformation
+import org.dataland.datalandbackend.openApiClient.model.CompanyInformation
 import org.dataland.datalandbackend.openApiClient.model.DataMetaInformation
-import org.dataland.datalandbackend.openApiClient.model.PostCompanyRequestBody
+import org.dataland.datalandbackend.openApiClient.model.StoredCompany
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertTrue
 import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDate
 
 class CompanyDataControllerTest {
 
-    private val basePathToBackendViaProxy = "http://proxy:80/api"
-    private val metaDataControllerApi = MetaDataControllerApi(basePathToBackendViaProxy)
-    private val companyDataControllerApi = CompanyDataControllerApi(basePathToBackendViaProxy)
-    private val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePathToBackendViaProxy)
+    private val basePathToDatalandProxy = "http://proxy:80/api"
+    private val metaDataControllerApi = MetaDataControllerApi(basePathToDatalandProxy)
+    private val companyDataControllerApi = CompanyDataControllerApi(basePathToDatalandProxy)
+    private val euTaxonomyDataControllerApi = EuTaxonomyDataControllerApi(basePathToDatalandProxy)
+    private val testCompanyList1 = listOf(
+        CompanyInformation(
```

Comment:
Die zwei Listen scheinen mir redundant. Man könnte die Tests unten doch auch mit nur einer durchführen, oder nicht?

Reply:
_Beginne es mir anzuschauen_

Reply:
Wir sind in den e2e-Tests immer den Weg gegangen, nie zweimal dieselbe Company zu posten.
Ich fands das auch immer ganz nett, weil man nach den Tests die vorliegenenden Company-Namen im CompanyStore den einzelnen Tests zuordnen konnte.

Da wir im CompanyDataController dreimal eine einzelne Company posten, und einmal eine Liste von Companies posten, brauchen wir eine Liste mit 3 Companies, und eine mit 2 anderen Companies.

Was ich aber gut finde, wäre es, das in eine eigene Klasse zu verfrachten. Z.B. in den `DummyDataCreator`.

Reply:
_Beginne mit Verfrachtung der Dummy-Company Erstellung in DummyDataCreator Klasse_